glusterd2/volume/metadata.go

```diff
@@ -7,4 +7,6 @@ const (
 	BlockHostingVolumeAutoCreated = "block-hosting-volume-auto-created"
 	// BlockHosting is a volume metadata which will be set as `yes' for volumes which are able to host block devices.
 	BlockHosting = "block-hosting"
+	// BlockPrefix is the prefix of the volume metadata which will contain BlockPrefix + blockname as the key and size of the block as value.
+	BlockPrefix = "block-vol:"
```

Comment:
will it work if we have both gluster-block and loopback volumes on the same BHV?

Reply:
Yeah, as in, the provider virt-block will store the metadata, not the gluster-block. Gluster-block can also be modified to use this, but doesn't do it at the moment.